menyer/for_goodness_shake/style.css

```diff
@@ -0,0 +1,98 @@
+body {
+    background-color: wheat;
+    text-align: center;
+    font-family: 'Architects Daughter', cursive;
+    max-width: 720px;
+    margin: auto;
+}
+
+.container {
+    position: relative;
+    display: flex;
```

Comment:
The assignment specifies to not use flex or grid.

Reply:
It no longer has flex or grid.